### ISSUE TITLE
Closing arrow barely visible in accessibility table view

### ISSUE THREAD

**Issue by deadkidney** [Problematy/goodmap-frontend#106]
In accessibility table view the closing arrow in left top corner is light gray on white background and small and therefore not visible at first glance.

![Image](https://github.com/user-attachments/assets/885a9591-1651-48e5-9700-5706930cf3a3)
